tensorflow/workspace.bzl

```diff
@@ -89,7 +89,7 @@ def tf_repositories(path_prefix = "", tf_repo_name = ""):
     dml_repository(
         name = "dml_redist",
         package = "Microsoft.AI.DirectML.Preview",
-        version = "1.5.0-dev20210301",
+        version = "1.6.0-dev20210318",
```

Comment:
this change can now be reverted

Reply:
Yep, been writing an email the past 20 minutes and haven't resolved the merge conflict yet :b.